example/example.tf

```diff
@@ -0,0 +1,46 @@
+terraform {
+  required_version = ">= 0.15.1"
+  required_providers {
+    azurerm = {
+      source  = "hashicorp/azurerm"
+      version = "=2.58.0"
+    }
+  }
+}
+
+provider "azurerm" {
+  features {}
+}
+provider "azuread" {
+}
+
+provider "azurerm" {
+  features {}
+  subscription_id = "ed302caf-ec27-4c64-a05e-85731c3ce90e"
+  alias           = "dnszones"
+}
+
+provider "azurerm" {
+  features {}
+  subscription_id = "1497c3d7-ab6d-4bb7-8a10-b51d03189ee3"
+  alias           = "privatezones"
```

Comment:
what is this provider for?

Reply:
this is required for cname entry. As i removed resource, i will delete this code as well.